plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/BridgeVifDriver.java

```diff
@@ -65,10 +65,7 @@ public void configure(Map<String, Object> params) throws ConfigurationException
 
         String networkScriptsDir = AgentPropertiesFileHandler.getPropertyValue(AgentProperties.NETWORK_SCRIPTS_DIR);
 
-        String controlCidr = (String)params.get("control.cidr");
-        if (StringUtils.isNotBlank(controlCidr)) {
-            _controlCidr = controlCidr;
-        }
+        _controlCidr = getControllCidr(_controlCidr, params);
```

Comment:
Single 'l' instead of 'll' for control?

Reply:
I think the double l is correct English (you dutchy;)

Reply:
I'd still say one l because that's how it's usually written and in the code right now everywhere